ui/components/forms/flavors/select/index.scss

```diff
@@ -22,3 +22,35 @@
     }
   }
 }
+
+#{$css-prefix}form-element__select-container {
```

Comment:
You are correct when it comes to standard BEM but we have some special rules in place that make this ok. We'll be releasing these patterns to the contributing guidelines shortly.